lib/share.js

```diff
@@ -68,8 +68,11 @@ module.exports = function sync (type, opts, dat) {
       // File Imports
       progressOutput[2] = 'Importing new & updated files to archive...'
 
-      // TODO: allow live: true
-      importStatus = dat.importFiles({live: false, resume: true}, function (err) {
+      importStatus = dat.importFiles({
+        live: false, // TODO: allow live: true. opts.live is used by archive.live though =(
```

Comment:
Yep `opts.live` means to watch the files (also means `archive.live` has to be true). 

We should probably change importer option to `watch` to avoid confusion.